obsbot/cogs/public/factoids.py

```diff
@@ -16,7 +16,8 @@ def __init__(self, bot, config):
         self.alias_map = dict()
         self.factoids = dict()
         self.config = config
-        self.limiter = RateLimiter(self.config.get('cooldown', 20.0))
+        self.member_limiter = RateLimiter(self.config.get('cooldown', 20.0))
+        self.channel_limiter = RateLimiter(self.config.get('cooldown', 20.0))
 
```

Comment:
Why are you adding a second instance of the rate limiter?

Reply:
One thing ping spammers do is run the same factoid with pings in every channel in quick order. This effectively prevents that, without putting any notable further limits on normal users. Basically, it only affects people that are doing something wrong.

Reply:
Yes, but the limiter is "stupid" in that it basically just checks for the key of whatever you use as input in its internal dict, so you don't need separate ones unless you want separate cooldowns.